### DISCUSSION TITLE
How to download beta?

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#2130] [Help] [answered]
I've been looking at testing for a particular issue I **might** encounter with 2.6, so I thought I'd download the beta, but I can't find a download link for it.

How can I download 2.6 beta 1?

EDIT: found it. It can be found [here](https://novelwriter.io/releases/release_2_6.html)

**Comment by vkbo** [comment, marked as answer]
It's also under [releases](https://github.com/vkbo/novelWriter/releases) on GitHub.
